packages/core/src/navigation/navigation-item.element.scss

```diff
@@ -9,20 +9,32 @@
 :host {
   --color: inherit;
   --line-height: inherit;
+  --focus-width: #{$cds-alias-object-border-width-300};
   --font-size: inherit;
   --font-weight: inherit;
   --letter-spacing: inherit;
   --padding: inherit;
+  padding-left: var(--focus-width);
+  border: var(--cds-alias-object-border-width-100) solid transparent;
 }
 
 :host(:hover:not([disabled])),
 :host([_has-focus]) {
   background: #{$cds-alias-object-interaction-background-hover};
+  border-color: var(--cds-alias-object-interaction-background-highlight);
```

Comment:
All the uses of `var(--cds-*)` should be updated to use the sass var `#{$cds-*}` so we get compile time checks

Reply:
Done